roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -194,6 +194,14 @@
 
 # storageclasses are used by default but if static then disable
 # storageclasses with the storageClassName set to "" in pvc.j2
+- name: Check to see if PVC already exists
+  oc_obj:
+    state: list
+    kind: pvc
+    name: "{{ openshift_logging_elasticsearch_pvc_name }}"
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+  register: logging_elasticsearch_pvc
+
```

Comment:
yeah... unfortunately `logging_elasticsearch_pvc.results.results | length > 0` returns a false positive... and the RC for being there and not being there is `0` in both cases. I could clean it up and move it into a block, it makes sense to me to do